### PR TITLE
Flex

### PR DESCRIPTION
**Pull request by SohaibCr7**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+    <head>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" type="text/css" href="stylesheet.css">
+    </head>
+    <body>
+        <div class="container">
```
Comment (SohaibCr7) [inline index.html:8]: change class name

File: index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                <p>This is  basic flex 2</p>
+
+            </div>
+            <div class="flex_basic3">
+                <p>This is  basic flex 3</p>
+
+            </div>
+        </div>
+        
+        <div class="container_direction">
```
Comment (SohaibCr7) [inline index.html:24]: qwerty

File: index.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+    <head>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" type="text/css" href="stylesheet.css">
```
Comment (adnanirfan) [inline index.html:5]: Add title tag

File: index.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+                <p>This is direction flex 3</p>
+
+            </div>
+
+        </div>
+
+        <div class="container_flex_flow">
+
+            <div class="flex_flow1">
+                <p>This is flex flow 1<br>Printing in Reverse</p>
```
Comment (adnanirfan) [inline index.html]: change `p` to `h2`

**Review by adnanirfan — CHANGES_REQUESTED** [review]

**Pull request merged**